pocs/observatory.py

```diff
@@ -537,6 +543,96 @@ def close_dome(self):
             self.logger.info('Closed dome')
         return self.dome.close()
 
+    def calibrate_mount(self, target=None, exp_time=30):
+        """Calibrates the mount to a target.
+
+        Note:
+            This assumes the mount is roughly polar aligned such
+            that a slew to a given AltAz will at the very least
+            point somewhere above the horizon.
+
+        Performs the following steps:
+             1. Slews to home.
+             2. Slews to target (default Alt=40° and Az=80°).
+             3. Takes a 30 second image with the primary camera.
+             4. Plates solves image and obtains coords for center of image.
+             5. Displays current separation.
+             6. Sets mount to current coords.
```

Comment:
Is there a point here where it waits for the user to do something with the mount alignment?